internal/kafka/internal/migrations/20220325111000_rename_eval_kafka_instance_type.go

```diff
@@ -0,0 +1,18 @@
+package migrations
+
+import (
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+func renameEvalKafkaInstanceType() *gormigrate.Migration {
+	return &gormigrate.Migration{
+		ID: "20220325111000",
+		Migrate: func(tx *gorm.DB) error {
+			return tx.Unscoped().Exec("UPDATE kafka_requests SET instance_type='developer' where instance_type='eval';").Error
+		},
+		Rollback: func(tx *gorm.DB) error {
+			return nil
```

Comment:
```suggestion
			return tx.Unscoped().Exec("UPDATE kafka_requests SET instance_type='eval' where instance_type='developer';").Error
```
Maybe the rollback will be the opposite.

Reply:
I thought the same. Btw. I think this for some reason doesn't work, e.g. when I have an `eval` kafka, its `instance_type` is not changed. do you know why it might be?

Reply:
The migration worked for me locally. How are you running the migration?

Reply:
I just started a service with `eval` kafka in the database. how did you run yours?

Reply:
I see. 
If you start the service with `make run`, it will execute the migrations for you if they weren't executed: there is not entry for `20220325111000` in the `migrations` table.
Alternative, if you run the binary, you could do:
```shell
./kas-fleet-manager migrate #run the migration
./kas-fleet-manager serve #run the service
```

Reply:
got it. thanks!